src/OpenApi/Model/Components.php

```diff
@@ -29,6 +29,8 @@ final class Components
 
     public function __construct(\ArrayObject $schemas = null, \ArrayObject $responses = null, \ArrayObject $parameters = null, \ArrayObject $examples = null, \ArrayObject $requestBodies = null, \ArrayObject $headers = null, \ArrayObject $securitySchemes = null, \ArrayObject $links = null, \ArrayObject $callbacks = null)
     {
+        !$schemas ?: $schemas->ksort();
```

Comment:
I would use a standard `if` for readability.

Reply:
@dunglas done.